mappings/1.8/net/minecraft/entity/ai/goal/WanderThroughVillageAtNightGoal.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/C_42bxzbove net/minecraft/entity/ai/goal/WanderThroughVillageAtNightGoal
 	FIELD f_03llfhsqx path Lnet/minecraft/C_58wymxilz;
-	FIELD f_21njodsls friendly Z
+	FIELD f_21njodsls isFriendly Z
```

Comment:
I think `friendly` is more consistent with conventions elsewhere in the mappings. Same goes for the arg below.